src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.scss

```diff
@@ -0,0 +1,11 @@
+.bc-lock-enable-label {
+  padding-top: 0;
+}
+
+.bc-lock-enable-help {
+  margin-left: 0.2em;
+}
+
+.bc-lock-enable-alert {
+  margin: 0 1em 0 0;
+}
```

Comment:
to keep things simplified, we usually go with [bootstrap spacing](https://getbootstrap.com/docs/5.0/utilities/spacing/).